gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri Nov 23 10:30:05 GMT 2018
```

Comment:
seems like this file keeps getting modified, I'm not sure why but I don't think it is breaking anything so happy to ignore this as not really related to this PR and we can think about it separately.

Reply:
Hmm... is it supposed to be checked in?